dsl/pace/dsl/dace/build.py

```diff
@@ -189,33 +111,31 @@ def set_distributed_caches(config: "DaceConfig"):
 
     # Execute specific initialization per orchestration state
     orchestration_mode = config.get_orchestrate()
+    if orchestration_mode == DaCeOrchestration.Python:
+        return
```

Comment:
What happened here?

Reply:
The system here is only for orchestration. "Python" mode, a.k.a stencil-mode, is not capable of taking the new system due to the fragmented JIT nature of the build.
I have a separate plan for this.